packages/terra-open-source-scripts/src/terra-cli/release/releaseRepoHandler.js

```diff
@@ -0,0 +1,41 @@
+const pacote = require('pacote');
+const path = require('path');
+const fs = require('fs-extra');
+const childProcess = require('child_process');
+const { promisify } = require('util');
+const Logger = require('@cerner/terra-cli/lib/utils/Logger');
+
+const setupGit = require('./setupGit');
+
+const logger = new Logger({ prefix: '[terra-open-source-scripts:release]' });
+
+const exec = promisify(childProcess.exec);
+
+const isPublished = async (name, version) => {
+  const pkgJson = await pacote.packument(name, {
+    registry: 'https://registry.npmjs.org/',
+  });
+  const publishedVersions = Object.keys(pkgJson.versions);
+  return publishedVersions.includes(version);
+};
+
+module.exports = async () => {
+  if (await isPublished()) {
+    logger.info('Nothing to publish');
+    return;
+  }
+  await fs.writeFile(path.join(process.env.HOME, '.npmrc'), `//registry.npmjs.org/:_authToken=${process.env.NPM_TOKEN}`, 'utf-8');
+
+  const packageFile = path.resolve(process.cwd(), 'package.json');
+
+  // Read package.json and pull out version
+  const { name, version } = await fs.readJSON(packageFile);
+
+  await setupGit(name, version);
```

Comment:
Can we make the order of publish, setupGit, tagging in git and pushing tags consistent between mono-repo and non-mono-repo packages?

Reply:
Done. https://github.com/cerner/terra-toolkit/pull/460/commits/6c35f4191a2115b9652c22dc9f42105554d0b08c